qiskit/tools/parallel.py

```diff
@@ -65,9 +65,6 @@
     # Default False on Windows
     if sys.platform == "win32":
         PARALLEL_DEFAULT = False
-    # On python 3.9 default false to avoid deadlock issues
-    elif sys.version_info[0] == 3 and sys.version_info[1] == 9:
-        PARALLEL_DEFAULT = False
```

Comment:
I think this is unrelated too (and at a glance it should presumably have been `sys.version_info[1] >= 9` anyway haha).

Reply:
Sigh, yeah that was also in my local experiment that I was playing with (the issue with python 3.9 is pickle would just occasionally hang when serializing a parameterized circuit, which was an issue for parallel transpile so I was trying to leverage qpy to workaround that). I'll fix this now too. I feel like I should just make a fresh checkout and start from scratch on this branch :)

Reply:
Ok, this time I hope I got it right, started from a fresh checkout and reran it